src/jobs/build_platform.yml

```diff
@@ -0,0 +1,21 @@
+description: >
+  Cross-build and store the binaries for a specific platform regexp.
+
+machine:
+  enabled: true
+
+parameters:
+  platform:
+    type: string
+    description: A promu platform regexp to build.
+
+steps:
+- setup_environment
+- run: promu crossbuild -v -p '<< parameters.platform >>'
+- persist_to_workspace:
```

Comment:
Let's remove this

Reply:
I want to keep this so I can do tests of matrix builds for the publishing. I don't know if it will work, but we need it to try. :smile:

Reply:
![image](https://user-images.githubusercontent.com/291750/111494802-13371b00-873f-11eb-9e00-7349155b060b.png)